src/main/java/net/fabricmc/example/mixin/ExampleMixin.java

```diff
@@ -1,15 +1,20 @@
 package net.fabricmc.example.mixin;
 
+import net.minecraft.client.gui.screen.Screen;
 import net.minecraft.client.gui.screen.TitleScreen;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
 @Mixin(TitleScreen.class)
-public class ExampleMixin {
+public class ExampleMixin extends Screen {
 	@Inject(at = @At("HEAD"), method = "init()V")
 	private void init(CallbackInfo info) {
 		System.out.println("This line is printed by an example mod mixin!");
 	}
+	@Inject(method = "render(IIF)V", at = @At("RETURN"))
```

Comment:
we should keep the element order consistent, so I think we should flip the `method` and `at` elements around.

Reply:
Putting method at first is actually the better practice because that allows you to find the injection target more swiftly.

Reply:
If we want to use that form for the examples, we should be consistent with it.

Reply:
Confirming we're moving to a `method, at` convention?